tests/compare_ncfile.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+import os
+import sys 
+import re
+import numpy as np
+##from __future__ import print_function
```

Comment:
You are not requiring python3 in the interpreter.  This line is necessary if `/usr/bin/env python` returns python version 2.